### PR TITLE
prebuilt ami: include subkey in image

### PR DESCRIPTION
**Pull request by mnaamani**
Include subkey installation when creating the prebuilt AMI image to save time when deploying new infrastructure.

### CODE REVIEW

**Comment by mnaamani** [comment]
It seems long running jobs started through ansible are problematic.  Probably needs similar treatment to the build step using async checks? @ahhda ?

File: devops/infrastructure/github-action-playbook.yml [hunk 1]
```diff
@@ -16,6 +16,11 @@
           name: common
           tasks_from: run-setup-build
 
+      - name: Install subkey
+        include_role:
+          name: admin
+          tasks_from: main
+
```
Comment (mnaamani) [inline devops/infrastructure/github-action-playbook.yml:23]: Also tried running subkey install in setup.sh https://github.com/Joystream/joystream/commit/52f0d99eafae6cb49c72c75454d3ca1d3a22c783

**Comment by ahhda** [comment]
> It seems long running jobs started through ansible are problematic. Probably needs similar treatment to the build step using async checks? @ahhda ?

Yes, looks like Github action do not go well with long running tasks. I have created a PR https://github.com/mnaamani/joystream/pull/8 which will resolve this issue.

**Comment by ahhda** [comment]
The latest action run was successful https://github.com/Joystream/joystream/actions/runs/1026331055

**Review by ahhda — APPROVED** [review]

**Pull request merged**